include/embedded_vagrantfile_xe_virtio

```diff
@@ -0,0 +1,70 @@
+# -*- mode: ruby -*-
```

Comment:
Just a comment- would be preferable imo if you could embed the logic in one vagrantfile - not introduce another. Just to avoid code-creep. If not possible I understand.

Reply:
At some point we may have just the one embedded vagrant file for XE, but right now I don't see how I can control the invocation of the `v.customize ['modifyvm', :id, '--nictype1', 'virtio']` that is required to ensure the VM comes up with a paravirtualized NIC. I could just make that the default? But I'd then need to go back and test pre-16.7.1 images for compatibility with paravirtualized NICs. For right now, I'd rather just leave it as an end-user choice, and hence have an alternate embedded vagrantfile if that's ok.